src/main/java/model/Code.java

```diff
@@ -0,0 +1,28 @@
+package model;
+
+public class Code {
+
+    private String code;
+    private Order order;
```

Comment:
maybe better write in order:
Code code;
and work with code in the order, because our order can't exist without code